modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempomonolithic-cli_{context}"]
+= Installing a TempoMonolithic instance by using the CLI
+
+:FeatureName: The TempoMonolithic instance
+include::snippets/technology-preview.adoc[]
+
+You can install a TempoMonolithic instance from the command line.
+
+.Prerequisites
+
+* An active {oc-first} session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your {oc-first} version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+.Procedure
+
+. Run the following command to create a project of your choice for the TempoMonolithic instance that you will create in a subsequent step:
```

Comment:
```suggestion
. Run the following command to create a project of your choice for the `TempoMonolithic` instance that you create in a subsequent step:
```

Reply:
Thank you for this suggestion. I only use back ticks when referring to the actual custom resource, which is not the case here.
Yes, I try to avoid future tense too. 🙂 However, "avoiding" and banning are not synonyms.